src/components/sbb-radio-button/sbb-radio-button.e2e.ts

```diff
@@ -0,0 +1,32 @@
+import { E2EElement, E2EPage, newE2EPage } from '@stencil/core/testing';
+
+describe('sbb-radio-button', () => {
+  let element: E2EElement, page: E2EPage;
+
+  beforeEach(async () => {
+    page = await newE2EPage();
+    await page.setContent('<sbb-radio-button value="Value">Value label</sbb-radio-button>');
+
+    element = await page.find('sbb-radio-button');
+  });
+
+  it('renders', async () => {
+    expect(element).toHaveClass('hydrated');
+  });
+
+  it('selects radio on click', async () => {
+    await element.click();
+    await page.waitForChanges();
+    expect(element).toHaveAttribute('checked');
+  });
+
+  it('does not deselect radio if already checked', async () => {
```

Comment:
can you add a test where deselection is included? (allowemptyselection)
and assert event is emitted in all tests